File: 0001 Code Chef/0014 Add two numbers/implementation/0010 nodejs/add_two_number.js
const readline = require('readline');

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

let test = 0;
let currentTest = 0;

rl.question('Enter the number of test cases: ', (input) => {
    test = parseInt(input);
    console.log(`Enter ${test} pairs of numbers:`);

    rl.on('line', (input) => {
        const stringArray = input.split(' ');
        const x = parseInt(stringArray[0]);
        const y = parseInt(stringArray[1]);
        console.log(x + y);

        currentTest++;
        if (currentTest === test) {
            rl.close(); // Close the readline interface once all test cases are processed
        }
    });
});

File: 0001 Code Chef/0014 Add two numbers/implementation/0003 c#/AddTwoNumber.cs
using System;

class AddTwoNumber{
    static void Main(string[] args){
        string stringInput = Console.ReadLine();
        int test = Convert.ToInt32(stringInput);
        while(test > 0){
            stringInput = Console.ReadLine();
            string[] stringArray = stringInput.Split(" ");
            int x = Convert.ToInt32(stringArray[0]);
            int y = Convert.ToInt32(stringArray[1]);
            Console.WriteLine(x+y);
            test = test - 1;
        }
    }
}